src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetProviderByUkprnQueryHandlerTests.cs

```diff
@@ -0,0 +1,32 @@
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.ProviderRegistrations.Data;
+using SFA.DAS.ProviderRegistrations.Models;
+using SFA.DAS.ProviderRegistrations.UnitTests.AutoFixture;
+using System.Threading;
+using System.Threading.Tasks;
+using SFA.DAS.ProviderRegistrations.Application.Queries.GetProviderByUkprn;
+
+namespace SFA.DAS.ProviderRegistrations.UnitTests.Application.Queries
+{
+    [TestFixture]
+    [Parallelizable]
+    public class GetProviderByUkprnQueryHandlerTests
+    {
+        [Test, ProviderAutoData]
+        public async Task Handle_WhenHandlingGetProviderByUkprnQueryAndParametersAreMatched_ThenShouldReturnGetAProviderNameResult(
+            long ukprn,
+            GetProviderByUkprnQueryHandler handler)
+        {
+            //arrange
+            var query = new GetProviderByUkprnQuery(ukprn);
+
+            //act
+            var result = await handler.Handle(query, new CancellationToken());
+
+            //assert
+            result.Should().NotBeNull();
+            result.ProviderName.Should().NotBeNullOrWhiteSpace();
```

Comment:
I'm not overly familiar with AutoData...but can we make the assertion more specific so it verifies the expected provider name is returned?

Reply:
No idea. Can we wait for Chris to return?

Reply:
Sure